h2o-core/src/main/java/water/rapids/Rapids.java

```diff
@@ -211,7 +214,7 @@ private AstFunction parseFunctionDefinition() {
   }
 
   /**
-   * Parse and return a list of tokens: either a list of strings, or a list of numbers.
```

Comment:
It only does numbers now, we removed the booleans

Reply:
Yeap.